### PR TITLE
SWS-91: Npm publishing

### PR DESCRIPTION
**Pull request by mtho11**
Publish the proper assets to npm

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "swsui",
-  "version": "0.1.0",
+  "version": "0.2.1",
```
Comment (jotak) [inline package.json]: already 0.2.1? Because you ran some test-purpose releases?

**Comment by jotak** [comment]
I guess you should rebase so that the storybook related commit is removed from this PR

**Comment by mtho11** [comment]
@jotak yes, I'll rebase

File: package.json (hunk 1 above)
Comment (mtho11) [reply package.json]: I had to get the proper files published to npm for mazz so it required several versions.
And also `@swift-sunshine/swsui` didn't work (paid feature only now) so I had to rollback to just `swsui`

**Comment by mtho11** [comment]
@jotak this should be good to go now

**Pull request merged**